env.yml

```diff
@@ -6,15 +6,6 @@ channels:
 dependencies:
   - appnope=0.1.0=py38_1001
   - backcall=0.2.0=py_0
-  - ca-certificates=2020.10.14=0
-  - certifi=2020.6.20=py38_0
-  - decorator=4.4.2=py_0
-  - ipython=7.19.0=py38h01d92e1_0
```

Comment:
I would much appreciate it if you could write some instructions of how you update your conda yml. That would be helpful